lib/galaxy/dependencies/dev-requirements.txt

```
@@ -193,8 +193,7 @@ pytz==2021.3; python_version >= "3.6" and python_full_version < "3.0.0" or pytho
 pyuwsgi==2.0.19.1.post0
 pyyaml==6.0; python_version >= "3.6"
 pyzmq==22.3.0; python_version >= "3.6"
-rdflib-jsonld==0.5.0; python_version >= "3.6" and python_version < "4"
-rdflib==4.2.2; python_version >= "3.6" and python_version < "4"
+rdflib==5.0.0; python_version >= "3.6" and python_version < "4"
```

```suggestion
rdflib==5.0.0
```
Since Python3.6 is your minimum, yes?

---

That's stuff added by poetry, it's not productive to fight it :smile:

---

Got it! I didn't know which was you and which was autogenerated

---

`pyproject.toml` is what we change manually (new requirements, pins...).